pages/role/apply/index.js

```diff
@@ -77,8 +77,11 @@ module.exports = settings => {
       const pelOrNprcTasks = res.locals.static.pelhOrNprcTasks;
       const rolesRequested = addRoleTasks.map(task => task.type).concat(pelOrNprcTasks.length > 0 ? PELH_OR_NPRC_ROLES : []);
 
+      const FEATURE_FLAG_NAMED_PERSON_MVP = 'feature-named-person-mvp';
```

Comment:
You should be able to import this as a constant from `asl-service` so that find usages will find it.